.github/workflows/cloudflare-open-next-build.yml

```diff
@@ -0,0 +1,69 @@
+# Security Notes
+# Only selected Actions are allowed within this repository. Please refer to (https://github.com/nodejs/nodejs.org/settings/actions)
+# for the full list of available actions. If you want to add a new one, please reach out a maintainer with Admin permissions.
+# REVIEWERS, please always double-check security practices before merging a PR that contains Workflow changes!!
+# AUTHORS, please only use actions with explicit SHA references, and avoid using `@master` or `@main` references or `@version` tags.
+
+name: Cloudflare OpenNext Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request_target:
```

Comment:
We don't use secrets in this workflow, so it can run on pull_request

Reply:
you're right, sorry I just copy-pasted this part without noticing this 👍